isso/css/isso.css

```diff
@@ -24,11 +24,8 @@
     vertical-align: bottom;
 }
 #isso-thread .isso-textarea {
-    min-height: 58px;
-    outline: 0;
-}
-#isso-thread .isso-textarea.isso-placeholder {
-    color: #757575;
+    width: 100%;
+    resize: none;
```

Comment:
The `outline: 0` should stay, no? Not that I'm opposed to changing that, but any styling changes should go into another PR.

Reply:
I'll add it back, but I can't really tell the difference. But you're right, it can be looked at later in a CSS cleanup.